code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -52,14 +52,25 @@
 	required_catalysts = list(/datum/reagent/stabilizing_agent = 1)
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_UNIQUE
 
+#define MAX_EXO_TEMP 1064
+
 /datum/chemical_reaction/plasma_solidification
 	required_reagents = list(/datum/reagent/iron = 5, /datum/reagent/consumable/frostoil = 5, /datum/reagent/toxin/plasma = 20)
```

Comment:
This breaks material conservation, should be 4 plasma instead of 20 as coin amount is 0.2 of sheet amount